x/ecocredit/server/marketplace/query_sell_order.go

```diff
@@ -0,0 +1,22 @@
+package marketplace
+
+import (
+	"context"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	v1 "github.com/regen-network/regen-ledger/x/ecocredit/marketplace"
```

Comment:
No need to use `v1` here. `marketplace` is more readable and will be easier to update.